test/rpc/test_rpc_misc.py

```diff
@@ -165,14 +165,13 @@ def test_get_events(input_data, expected_data):
             account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
             private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
         )
-    resp = rpc_call("starknet_getEvents", params=input_data)
+    resp = rpc_call("starknet_getEvents", params=input_data["params"])
     assert len(expected_data) == len(resp["result"]["events"])
     for i, data in enumerate(expected_data):
         assert str(resp["result"]["events"][i]["data"]) == str(data)
 
-    if "continuation_token" in input_data:
-        expected_continuation_token = int(input_data["continuation_token"])
-
+    if "continuation_token" in input_data["params"]["filter"]:
```

Comment:
What if the continuation_token is not present but it is expected to be?

Reply:
Let's say someone will remove `continuation_token` for these test calls. Then `continuation_token` will be set to `0` as the default value in `get_events` function and firsts events will be returned and one test will fail because `expected_data` will be different. 

![image](https://user-images.githubusercontent.com/5063930/213999310-1712a995-0e35-4703-81d5-36e88d64b375.png)
Just tested it

Reply:
I slightly understand (not completely), but I'll believe you on this one. Marking as resolved.